WORKSPACE

```diff
@@ -58,12 +58,12 @@ http_archive(
 )
 
 ##### GRPC
-_GRPC_VERSION = "1.42.0"
+_GRPC_VERSION = "1.48.0"
```

Comment:
Can you add a comment that points to the version referenced: https://github.com/grpc/grpc/releases/tag/v1.48.0